content/en/community/end-user/slack-channel.md

```diff
@@ -6,21 +6,17 @@ description: >-
 weight: 40
 ---
 
-Previously, we had a private channel to provide a space for end users to discuss
-adoption and implementation issues while allowing them to maintain a layer of
-privacy and to meet compliance concerns. Now that the project and community have
-grown, and with the transition of the End User WG to End User SIG, we believe
-that this private channel is no longer needed.
+Join our Slack channel to connect with other OpenTelemetry end users! First, 
+[request an invitation](https://communityinviter.com/apps/cloud-native/cncf) 
+to CNCF's Slack instance.
 
```

Comment:
```suggestion
Join our Slack channel to connect with other OpenTelemetry end users! 

First, [request an invitation](https://communityinviter.com/apps/cloud-native/cncf) 
to CNCF's Slack instance. Next, join [`#otel-sig-end-user`](https://cloud-native.slack.com/archives/C01RT3MSWGZ), and introduce yourself if you'd like. 
```

Reply:
I messed up and wasn't able to directly commit this suggestion, but have added it manually to the file.